documentation/source/install.rst

```diff
@@ -0,0 +1,117 @@
+.. _install:
+
+************
+Installation
+************
+
+Using cocotb requires :ref:`installation of pre-requisites<install-pre-requisites>` and
+:ref:`installation of cocotb<installation-via-pip>` itself.
+In this document, we are assuming that you already have a
+:ref:`supported simulator<simulator-support>` available in ``PATH``.
+
+.. _install-pre-requisites:
+
+Pre-requisites
+==============
+
+Cocotb has the following requirements:
+
+* Python 3.5+
+* Python-dev packages
+* GCC 4.8.1+ or Clang 3.3+ and associated development packages
+* GNU Make
+* A Verilog or VHDL simulator, depending on your RTL source code
+
+.. versionchanged:: 1.4 Dropped Python 2 support
+
+.. note:: In order to use a 32-bit simulator you need to use a 32-bit version of Python.
+
+
+Windows
+-------
+
+`Conda <https://conda.io/>`_ is an open-source package and environment management system that we recommend for Windows.
+
+Download and install `Miniconda <https://docs.conda.io/en/latest/miniconda.html>`_ from https://conda.io/.
+From a Command Prompt or the Windows PowerShell, use the following line to install a compiler (GCC or Clang) and GNU Make:
+
+.. code-block:: bash
+
+    conda install -c msys2 m2-base m2-make m2w64-toolchain libpython
+
```

Comment:
```suggestion

.. note:: 

    In case errors related to ``Microsoft Visual Studio`` you need to set MinGW as the default compiler details can be found `here <https://stackoverflow.com/questions/3297254/how-to-use-mingws-gcc-compiler-when-installing-python-package-using-pip>`_.

```